Documentation/community.rst

```diff
@@ -17,10 +17,10 @@ and meeting links.
 ====================== ===================================== ============= ================================================================================
 SIG                    Meeting                               Slack         Description
 ====================== ===================================== ============= ================================================================================
-Datapath               Every 2 weeks                         #sig-datapath Owner of all BPF and Linux kernel related datapath code.
+Datapath               Wednesdays, 08:00 PT                  #sig-datapath Owner of all BPF and Linux kernel related datapath code.
 Documentation          None                                  #sig-docs     All documentation related discussions
-Envoy                  Every 2 weeks                         #sig-envoy    Envoy, Istio and maintenance of all L7 protocol parsers.
-Policy                 Every Wed, 9:30 PT (`Policy-Zoom`_)   #sig-policy   All topics related to policy. The SIG is responsible for all security relevant APIs and the enforcement logic.
+Envoy                  Biweekly on Thursdays, 09:00 PT       #sig-envoy    Envoy, Istio and maintenance of all L7 protocol parsers.
```

Comment:
Not sure of my English here, doesn't “biweekly” mean “twice a week”?

Reply:
It can mean both according to https://en.wikipedia.org/wiki/Biweekly :smile: Probably better to use an unambiguous term then?

Reply:
Given that it's `Biweekly on Thursdays`, isn't it unambiguous already?

Reply:
Right, I only read @qmonnet's comment, not the original change 😄 might still be less ambiguous to say `Thursday every other week` or `every other Thursday`? I'm not a native English speaker though, so I'll leave for others to judge.

Reply:
I did a double-take when I wrote this too, because yes, in english it can mean either.

Given the "biweekly" + "on thursdays" and that the typical week only includes a single thursday, I think it's probably clear enough to readers.

In the surrounding text, we suggest people to join the slack channel for the relevant SIG to check the actual dates of the next meeting so they should be able to figure it out from there.

Reply:
> the typical week only includes a single thursday

{{Citation needed}}

Okay, fair enough :)